V2 failed

Running the V2 action, got the following error:

```
6798510c0a2d: Pull complete
eb5cadd8df76: Pull complete
491b99df4756: Pull complete
3de2608fb972: Pull complete
803c70a80654: Pull complete
1fe51443aaa3: Pull complete
c818c0dbc053: Pull complete
8d2da93fd434: Pull complete
Digest: sha256:a9a65de19ab6108f6b5d58b3fe6cfc40cc0bbef44d4c2b5689aaa1e1c6bba1d5
Status: Downloaded newer image for spicelabs/spice-labs-cli:latest
Unknown option: '--verbose'
Possible solutions: --version
Error: Process completed with exit code 2.
```

Maybe tie versions of the action to versions of the cli container?

--- a/action.yml
+++ b/action.yml
@@ -25,7 +25,7 @@ runs:
         docker run \
           -v "${{ inputs.file_path }}:/mnt/input:ro" \
           -e SPICE_PASS="${{ inputs.spice_pass }}" \
-          spicelabs/spice-labs-cli:latest \
+          spicelabs/spice-labs-cli:0.2 \
           --log-level all \
           --input /mnt/input \
           --tag "${{ inputs.tag }}"

